app/controllers/orders/active_items.js

```diff
@@ -1,29 +1,74 @@
 import detail from "./detail";
+import {
+  ORDER_SORTING_OPTIONS,
+  ORDER_PACKAGES_STATES
+} from "stock/constants/order-sorting-options";
+import SearchMixin from "stock/mixins/search_resource";
 import _ from "lodash";
 
-export default detail.extend({
+export default detail.extend(SearchMixin, {
+  // ----- Services -----
   goodcityRequestService: Ember.inject.service(),
   packageService: Ember.inject.service(),
   packageTypeService: Ember.inject.service(),
 
+  // ----- Arguments -----
+  dropDownItems: ORDER_SORTING_OPTIONS,
+  states: ORDER_PACKAGES_STATES,
+  displayDropDownItems: false,
+  sortingColumn: ORDER_SORTING_OPTIONS[0],
   autoLoad: true,
   /*
    * @type {Number}, perPage in response
    */
   perPage: 25,
   ordersPkgLength: 0,
   sortProperties: ["id"],
+
+  // ----- Computed Properties -----
   sortedGcRequests: Ember.computed.sort(
     "model.goodcityRequests",
     "sortProperties"
   ),
 
+  filteredStates: Ember.computed("states.@each.enabled", function() {
+    this.reloadResults();
+    return this.getFilteredStates();
```

Comment:
```suggestion
    return _.filter(this.get("states"), ["enabled", true]).map(
      ({ state }) => state
    );
```

You can put the body straight in, and remove the `getFilteredStates` method